larq_zoo/datasets.py

```diff
@@ -0,0 +1,30 @@
+from zookeeper import Field, component
+from zookeeper.tf import TFDSDataset
+
+
+@component
+class ImageNet(TFDSDataset):
+    name = Field("imagenet2012:5.0.*")
+    train_split = Field("train")
+    validation_split = Field("validation")
+
+
+@component
+class Cifar10(TFDSDataset):
```

Comment:
We could move all of these to zookeeper too, no?

Reply:
I do broadly agree with you on this, but I think it falls into the same bucket as the `ModelFactory` class - something that we can maybe move later.